src/components/SidePanel.js

```diff
@@ -147,8 +147,4 @@ SidePanel.propTypes = {
     ]),
 };
 
-SidePanel.defaultProps = {
```

Comment:
If we are removing it from here, please add it as default value in the component parameter so that storybook renders the default value in the docs.

Reply:
Updated default value for SidePanel.